.github/workflows/tests.yml

```
@@ -7,7 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         python: [3.7, 3.8, 3.9]
-        plone: ["6.0.0a6", "5.2-latest"]
+        plone: ["6.0.0b1", "5.2-latest"]
```

Let's just use 6.0-latest here and then we don't need to remember to update it when there is a new release
```suggestion
        plone: ["6.0-latest", "5.2-latest"]
```